### DISCUSSION TITLE
Loop to create row creates an error

### DISCUSSION THREAD

**Discussion by guyaldous1** [Budibase/budibase#14108] [Help] [answered]
I have a database query that returns an array, ive included the output as a table below for reference. 
<img width="882" alt="Screenshot 2024-07-05 at 12 50 12 pm" src="https://github.com/Budibase/budibase/assets/39635882/1d24dbdc-ee8c-4268-8e9b-31befbfa6dfe">
I'd like to push each row of this table to a bubibase DB. I've created a loop on create row and I continually get an error when testing. 
<img width="806" alt="Screenshot 2024-07-05 at 1 45 36 pm" src="https://github.com/Budibase/budibase/assets/39635882/3e865902-4e2d-4610-ba03-74390a6723d5">

I've connected the same loop and query to a slack messaging function to test and it works fine in that context.
<img width="704" alt="Screenshot 2024-07-05 at 1 45 46 pm" src="https://github.com/Budibase/budibase/assets/39635882/703b6b2b-25be-4371-a157-035d576fd50f">

Would you be able to point me in the right direction if I'm doing something wrong.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @guyaldous1,

This issue was fixed recently. I'm assuming you are on a older version of Budibase, before the fix was released. We are currently on `2.29.6`.

I hope this helps.
